src/widgets/standardGoalsList.js

```diff
@@ -0,0 +1,70 @@
+import { REPORT_STATUSES } from '@ttahub/common';
+import { Op } from 'sequelize';
+import {
+  ActivityReport,
+  ActivityReportGoal,
+  Goal,
+  GoalTemplate,
+  sequelize,
+} from '../models';
+import { CREATION_METHOD } from '../constants';
+
+export default async function standardGoalsList(scopes) {
+  // Query for all standard goals linked to activity reports
+  const results = await ActivityReport.findAll({
+    attributes: [
+      [sequelize.col('activityReportGoals.goal.goalTemplate.id'), 'goalTemplateId'],
+      [sequelize.col('activityReportGoals.goal.goalTemplate.standard'), 'standard'],
+      [sequelize.fn('COUNT', sequelize.literal('DISTINCT "ActivityReport"."id"')), 'count'],
+    ],
+    where: {
+      calculatedStatus: REPORT_STATUSES.APPROVED,
+      startDate: {
+        [Op.gte]: new Date('2025-09-01'),
+      },
+      ...scopes.activityReport,
+    },
+    include: [
+      {
+        model: ActivityReportGoal,
+        as: 'activityReportGoals',
+        attributes: [],
+        required: true,
+        include: [
+          {
+            model: Goal,
+            as: 'goal',
+            attributes: [],
+            required: true,
+            /* where: {
+              prestandard: false,
+            }, */
```

Comment:
remove if not needed

Reply:
We need this but I'm commenting it out to get more records back.